docs/developer/Releases.md

```diff
@@ -0,0 +1,116 @@
+Release Approach
+================
+
+## Table of Contents
+
+* [Versioning](#versioning)
+  * [API Compatibility](#api-compatibility)
+  * [Modules Providing API](#modules-providing-api)
+  * [Towards a First Release](#towards-a-first-release)
+* [Legal Documentation Requirements](#legal-documentation-requirements)
+  * [License and Notice Files](#license-and-notice-files)
+  * [Creating the Notice File](#creating-the-notice-file)
+  * [Background Information](#background-information)
+* [Publishing Maven Artifacts](#publishing-maven-artifacts)
+  * [Naming Convention](#naming-convention)
+
+### Versioning
+
+The Eclipse Dataspace Connector employs [SemVer](https://semver.org) for versioning and distinguishes between the following releases as defined by the [Eclipse Handbook](https://www.eclipse.org/projects/handbook/#release):
+
+- Major releases, which introduce API changes,
+- Minor releases, which add new functionality, but are API compatible with previous versions, and
+- Service releases, which include bug fixes only and add no significant new functionality.
+
+Between releases, snapshot versions reflecting the current state of modules can be packaged to distribution artifacts on a regular basis. 
+Snapshots, however, do not actually represent released versions.
+
+#### API Compatibility
+
+The concept of API compatibility is defined in terms of binary compatibility according to the [Java SE 11 Language Specification](https://docs.oracle.com/javase/specs/jls/se11/html/jls-13.html) 
+
+#### Modules Providing API
+
+The following modules specify a Java Service Provider Interface (SPI) and contribute public classes, interfaces and public members which are considered public APIs and are covered by the proposed versioning approach:
```

Comment:
To be precise, I would say "The following modules define official extension points for the EDC and contribute..."